### PR TITLE
Research Reactor

### PR DESCRIPTION
**Pull request by Vaern**
bunger

### CODE REVIEW

**Comment by The-Quarker** [comment]
burger bunker

**Comment by Vaern** [comment]
IMG_4522

File: src/main/java/com/hbm/blocks/ModBlocks.java [hunk 1]
```diff
@@ -1712,8 +1709,7 @@ private static void initializeBlock() {
 		
 		machine_puf6_tank = new MachinePuF6Tank(Material.iron).setBlockName("machine_puf6_tank").setHardness(5.0F).setResistance(10.0F).setCreativeTab(MainRegistry.machineTab);
 		
-		machine_reactor = new MachineReactor(Material.iron).setBlockName("machine_reactor").setHardness(5.0F).setResistance(10.0F).setCreativeTab(MainRegistry.machineTab).setBlockTextureName(RefStrings.MODID + ":machine_reactor");
-		machine_reactor_on = new MachineReactor(Material.iron).setBlockName("machine_reactor_on").setHardness(5.0F).setResistance(10.0F).setCreativeTab(null).setBlockTextureName(RefStrings.MODID + ":machine_reactor_on");
+		machine_reactor_breeding = new MachineReactorBreeding(Material.iron).setBlockName("machine_reactor_breeding").setHardness(5.0F).setResistance(10.0F).setCreativeTab(MainRegistry.machineTab).setBlockTextureName(RefStrings.MODID + ":machine_reactor");
```
Comment (HbmMods) [inline src/main/java/com/hbm/blocks/ModBlocks.java]: image
Hold on there buster, you're about to pulverize everyone's breeding reactors

Comment (Vaern) [reply src/main/java/com/hbm/blocks/ModBlocks.java]: We already discussed this
if it's too much of an issue you can just change it back once it's merged

Comment (UFFR) [reply src/main/java/com/hbm/blocks/ModBlocks.java]: ran fumo spin

Comment (Vaern) [reply src/main/java/com/hbm/blocks/ModBlocks.java]: scheiße

File: src/main/java/com/hbm/hazard/HazardRegistry.java [hunk 2]
```diff
@@ -191,42 +193,49 @@ public static void registerItems() {
...
+		registerOtherFuel(plate_fuel_pu239, pu239 * ingot, pu239 * ingot * 100);
+		registerOtherFuel(plate_fuel_sa326, sa326 * ingot, sa326 * ingot * 100);
+		registerOtherFuel(plate_fuel_ra226be, rabe * ingot, po210 * nugget * 3);
+		
+		registerOtherWaste(waste_plate_u233, u233 * ingot * 100);
+		registerOtherWaste(waste_plate_u235, u235 * ingot * 100);
+		registerOtherWaste(waste_plate_mox, mox * ingot * 100);
+		registerOtherWaste(waste_plate_pu239, pu239 * ingot * 100);
+		registerOtherWaste(waste_plate_sa326, sa326 * ingot * 100);
+		registerOtherWaste(waste_plate_ra226be, po210 * billet * 3);
```
Comment (Vaern) [inline src/main/java/com/hbm/hazard/HazardRegistry.java]: Oops, this should be * nugget, not * billet

File: src/main/java/com/hbm/packet/AuxGaugePacket.java [hunk 3]
```diff
@@ -23,7 +23,7 @@
 import com.hbm.tileentity.machine.TileEntityMachineReactorLarge.ReactorFuelType;
 import com.hbm.tileentity.turret.TileEntityTurretCIWS;
 import com.hbm.tileentity.turret.TileEntityTurretCheapo;
-import com.hbm.tileentity.machine.TileEntityMachineReactorSmall;
+import com.hbm.tileentity.machine.TileEntityReactorResearch;
```
Comment (Vaern) [inline src/main/java/com/hbm/packet/AuxGaugePacket.java]: ach
my ide doesn't show me unused imports anymore

**Pull request merged**